generators/app/templates/webpack.config.js

```diff
@@ -23,6 +23,7 @@ if (process.env.INCLUDE_WEBPACK_HTML) {
 
 module.exports = {
   entry: [
+    <%- withFetch ? "'whatwg-fetch'," : '' %>
```

Comment:
Boom, got it. Looks a little elementary but works and doesn't leave that blank line.